### DISCUSSION TITLE
Was the shortcut key title intentionally removed in 0.8.0?

### DISCUSSION THREAD

**Discussion by itsramiel** [okwasniewski/MiniSim#92] [Q&A] [answered]
After I upgraded to 0.8.0(as I remember) I no longer see the shortcut keys next to simulators, which was a **very helpful** feature as I like to use the keyboard instead of the mouse. The shortcuts still work, but they are visually not visible so I have to do some guessing.

Here is how it looks now:
<img width="401" alt="image" src="https://github.com/okwasniewski/MiniSim/assets/80689446/08bdd1b4-e8e0-418a-985f-13630898f166">

Here is how it looked before(screenshot is from the homepage of this github):
<img width="420" alt="image" src="https://github.com/okwasniewski/MiniSim/assets/80689446/506f6bcb-e76a-48bb-855f-a839b9e66025">

Is this a bug and I need to open an issue?

**Comment by okwasniewski** [comment, marked as answer]
Hey! Unfortunately, this is a change in macOS Sonoma which affects showing shortcuts for menu items with submenus. 

I've tweeted about it here: https://twitter.com/o_kwasniewski/status/1708405770188996634

As Apple says: "Menus were rewritten from the ground up" - this looks like a regression on their side. I will file bug report  for them and think of some workaround 👍🏻
